srcpkgs/zita-convolver/template

```diff
@@ -14,15 +14,17 @@ homepage="http://kokkinizita.linuxaudio.org/linuxaudio/"
 distfiles="http://kokkinizita.linuxaudio.org/linuxaudio/downloads/${pkgname}-${version}.tar.bz2"
 checksum=9aa11484fb30b4e6ef00c8a3281eebcfad9221e3937b1beb5fe21b748d89325f
 
-CXXFLAGS="-fPIC -ffast-math -funroll-loops"
+pre_build() {
+	CXXFLAGS="$CXXFLAGS -fPIC"
+}
```

Comment:
Why change this? Removing `-ffast-math` is fine tho.

Reply:
```
/usr/bin/ld: zita-convolver.o: relocation R_X86_64_PC32 against symbol `_ZN9Convlevel11static_mainEPv' can not be used when making a shared object; recompile with -fPIC
```

Reply:
https://github.com/void-linux/void-packages/pull/31666/checks?check_run_id=2925832556#step:8:60